src/components/EmailAddress.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  export let emails;
+</script>
+
+<style>
+  td {
+    @apply border;
+    @apply p-2;
+  }
+</style>
+
+<tr>
+  <td>Notification Email{emails.length > 1 ? 's' : ''}</td>
+  <td>
+    {#each emails as email}
+      <a href="mailto:{email}" target="_blank">{email}</a>
+      <br />
```

Comment:
Instead of using `<br/>` here (which is a bit hacky), I'd put them in a span where class="block" (see original code)